cdk/bin/cdk.ts

```diff
@@ -15,11 +15,13 @@ new Gatehouse(app, 'gatehouse-CODE', {
     env,
     stage: 'CODE',
     domainName: 'gatehouse-origin.code.dev-guardianapis.com',
+    rdsSecurityGroupId: 'sg-ce0783b5',
```

Comment:
why is the security group id hardcoded here ? In all other places where we don't use cdk it's an external paramater

Reply:
I'll read it in from a param instead

Reply:
That's probably why I hardcoded it!  But I'm sure it can be done

Reply:
Done in 757953a2a4f6d17d52938c598475e7c3d7abad26